ODBC/OAuthSampleApp.cpp

```diff
@@ -74,18 +74,10 @@ void connectToDB(
     SQLHDBC &hdlDbc,
     std::string accessToken,
     std::string refreshToken,
-    std::string clientId,
-    std::string clientSecret,
-    std::string tokenUrl,
-    std::string scope)
+    std::string clientSecret)
 {
     // Construct json config for static oauth config
-    std::string jsonConfig = std::string("{") +
-	"\"oauthtokenurl\" : \"" + tokenUrl + "\", " +
-        "\"oauthclientid\" : \"" + clientId + "\", " +
-        "\"oauthclientsecret\" : \"" + clientSecret + "\", " +
-        "\"oauthscope\" : \"" + scope + "\"" +
-        "}";
+    std::string jsonConfig = std::string("{\"oauthclientsecret\" : \"" + clientSecret + "\"}";
```

Comment:
I think you're missing a closing ')' here.

Reply:
Right you are.  Fixed.